scipy-notebook/Dockerfile

```diff
@@ -32,12 +34,15 @@ RUN conda install --quiet --yes \
     'dill=0.2*' \
     'numba=0.23*' \
     'bokeh=0.11*' \
-    'h5py=2.5*' \
-    && conda clean -tipsy
+    'h5py=2.5*' && \
+    conda remove --force qt pyqt && \
```

Comment:
Nah. It's my bad. I copy/pasted the command into the Python 2 environment setup, but forgot that I'll need to target that environment.